### ISSUE TITLE
Video Annotations feature request

### ISSUE THREAD

**Issue by ambustion** [nextcloud/files_videoplayer#137]
Is it possible to add a simple button for comments attached to timecode on videos in nextcloud? I know [this github videojs plugin](https://github.com/contently/videojs-annotation-comments) has a lot of functionality that would be great. This would be an incredible feature to have on self hosted storage for video creators and anyone collaborating on video in general. I've attempted adapting the files_videoplayer to tie in the javascript for this but I am out of my element getting it recognized.

**Comment by wedza** [comment]
This would definitely be an awesome feature for video producers. There are only very expensive proprietary solutions like frame.io, wipster and vimeo which offers this. This video.js plugin seems already very powerful.

**Comment by mzed2k** [comment]
Yes, this would be a cool feature, I'd like it to see that for audio files, too. Just some interaction between media player and nextcloud comments.

**Comment by netbrut** [comment]
Hi, I need too...

**Comment by liechtjc** [comment]
It would be really such a nice feature. I just deployed nextcloud to share videos to be reviewed and it's already awsome...

**Comment by ambustion** [comment]
I know the feature is currently implemented in pydio enterprise, and have seen a version of nextcloud with it, but it was a custom rolled version someone wanted to sell themselves. I would love to see it in there still.

**Comment by brendonmconnett** [comment]
This would be amazing. Loads of possibility with the simple ability for time stamp buttons and maybe a way for the uploader to check the timestamp comments off?

**Comment by liechtjc** [comment]
How could we achieve this? Offering a bounty for an App dev? 
We started a few month a Nextcloud connected to S3 storage in Wasabi to share video and it's close to be a killer app. We are pushing content in Webdav from the edit station. We still need to fix some timeout issues when you are uploading large files. I will also have a look at automated media conversion (knowing that it can consume a lot of cloud resource to transcode and that a single instance implementation could be a problem)...

**Comment by Noshire** [comment]
This would be a great tool - I find the annotations/timestamps very useful e.g. on Youtube, and would love to have that functionality on nextcloud as well!

**Comment by StephanRoemer** [comment]
Giving this a +1. Been looking for something like that.

**Comment by vgarlu** [comment]
Would love to see the video annotations implemented as well. +1

**Comment by matthewfu** [comment]
It would be great if video annotations implemented !

**Comment by aniketshiro** [comment]
Yes. This is would be a game changer for all video production professionals as it would reduce costs quite significantly. It would be really helpful especially if we could export the annotations and the timestamps in an EDL Format for DaVinci Resolve. Please let us know if we can support this project in anyway

**Comment by dpalic** [comment]
any updates on this?
Would be a really great feature. This feature is hindering me to switch from dropbox to nextcloud

**Comment by TheRealFreelancer** [comment]
I second that idea. It would be a gamechanger collecting commentary from clients for professional video production directly in the video.

**Comment by himanshu-ntml** [comment]
It would be such a great feature in addition to comments module. It will help in giving very direct feedback and will allow fast iteration when producing content. Thanks!

**Comment by daniel-lupi** [comment]
Yes, PLEASE!
